src/utils/redstone_tx_sync/client.go

```diff
@@ -0,0 +1,111 @@
+package redstone_tx_sync
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/sirupsen/logrus"
+	"github.com/warp-contracts/syncer/src/utils/bundlr"
+	"github.com/warp-contracts/syncer/src/utils/config"
+	"github.com/warp-contracts/syncer/src/utils/logger"
+	"github.com/warp-contracts/syncer/src/utils/model"
+	"github.com/warp-contracts/syncer/src/utils/sequencer"
+	sequencer_types "github.com/warp-contracts/syncer/src/utils/sequencer/types"
+	"gorm.io/gorm"
+)
+
+type Client struct {
```

Comment:
I think the whole client isn't needed. We could just use ethClient and sequencerClient

Reply:
You're right, it made more sense in the previous impl